diff --git a/sentry.gradle.kts b/sentry.gradle.kts
index 4d8762ccd6343d1f71f55eea967f96af80b887eb..d018de002c3d92254a7f91c5e7332667fe9c114c 100644
--- a/sentry.gradle.kts
+++ b/sentry.gradle.kts
@@ -266,6 +266,7 @@ fun extractBundleTaskArguments(
     val intermediateSourcemapsDirFile =
         when (jsIntermediateSourceMapsDir) {
             is org.gradle.api.file.Directory -> jsIntermediateSourceMapsDir.asFile
+            is org.gradle.api.file.RegularFile -> jsIntermediateSourceMapsDir.asFile
             else -> return BundleTaskArgs(null, null, null, null)
         }
 
